src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <section class="container">
+      <div class="calendar calendar--month-length-31 calendar--start-day-sun">
+        <div class="calendar__day-1"></div>
```

Comment:
you must not use number with calendar-day, use nth-child to calculate target
```suggestion
        <div class="calendar__day"></div>
```